decorator.rb

```diff
@@ -0,0 +1,14 @@
+require './nameable'
+
+class Decorator < Nameable
+  attr_accessor :nameable
+
+  def initialize(nameable)
+    super()
```

Comment:
- Optional
Since you are not passing anything to the parent constructor, there is not point in calling `super()` here. You can consider removing it.

Reply:
Hi @muneebulrehman, but I don't understand why Linters insists on adding a super in this case.
Rubocop msg: "Call `super` to initialize state of the parent class. (warning:Lint/MissingSuper)"